docs/source/image.rst

```diff
@@ -0,0 +1,17 @@
+================
+Image Processing
+================
+
+Image
+=====
+.. automodule:: lcls_tools.common.image.image 
+    :members:
+    :undoc-members:
+
+.. automodule:: lcls_tools.common.image.processing
+    :members:
+    :undoc-members:
+
+.. automodule:: lcls_tools.common.image_processing.roi
+    :members:
```

Comment:
Is image_processing still needed with the rename?

Reply:
Would this be image.processing.roi?

Reply:
Yes it would! Good catch.